<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="AWS-PVDI-25" failures="1" tests="1" name="Default test" time="83.217" errors="0" timestamp="2023-12-12T12:23:56 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="68.996">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [In-active] but found [Active],
	expected [Active] but found [In-active]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [In-active] but found [Active],
expected [Active] but found [In-active]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefinitions.AutoLoanAppdataEntryContactDetails.user_076_invoke_soft_assert_in_contac_details_new_new_app_stage(AutoLoanAppdataEntryContactDetails.java:1588)
at âœ½.user_076 invoke softAssert in contac details new new app stage(file:///C:/Users/inindc00607/git/ULS_Ijarah_1/Gowtham_ULS_New/src/test/java/features/AutoLoanAppdataEntryContactDetails.feature:94)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
